modules/javafx.controls/src/test/java/test/javafx/css/PropertySizeTest.java

```diff
@@ -216,6 +216,9 @@ public void relativeFontSizeSetOnAllNestedParentsAndControlsTest() {
         p3.setStyle("-fx-font-size: 0.6em");
         p4.setStyle("-fx-font-size: 0.5em");
 
+        // When a Parent and its Parent's font size is relative then that Parent's
+        // font size is computed in relative to it's grand parent's font size.
```

Comment:
Very mInor: grandparent is one word (I only mention in case there are other changes you are making at the same time).